src/utils/updateDBIcons.ts

```diff
@@ -0,0 +1,129 @@
+import axios from 'axios'
+import util from 'util'
+import cmd from 'node-cmd'
+import { iconsServices, IconInterface, IIconsModel } from 'components/icons'
+import { infoServices } from 'components/info'
+import { Logger, NodeLogger } from 'helpers'
+import { getEncodedLink, isNewIcon } from './tools'
+import prepareUpdatedIcon from './tools/updatingIcons/prepareTheUpdate'
+import { getFilled, getOutlined } from './tools/updatingIcons/getSvgCode'
+import configs from 'configs'
+// promise-style version of cmd.get
+const pcmdGet = util.promisify(cmd.get)
+
+const updateIconsLogger = new Logger('updateDBIcons')
+
+const updateDBIcons = async (notifiedByHook = false) => {
+  try {
+    // Latest Database version's info:
+    const currentVersionInfo = await infoServices.getLatestVersionInfo()
+    // in initial setup, there wouldn't be any versions stored in the db, that's why we set an old date.
+    const currVersionDate = new Date(currentVersionInfo ? currentVersionInfo.createdAt : '1999-03-20')
```

Comment:
This's a random `date` to be used when there's no versions at all in the `db`, to be able to fetch all the icons, in the initialization of the db phase. 

it's not random tbh, it's my bd :p. Anyway, it can be any old date.

Reply:
Why not use the current date to know when it was populated?

Reply:
I don't know how I missed this comment! sorry for the late reply @kbhutani0001. 

if I understood your question correctly, you mean using `new Date()` instead of `1999-03-20` right? 
We will only use `1999-03-20` in case, if there's no info records in the db (server is initially configured), it means that we must fetch all the icons, regardless of it's date, that's why I've used a very old date `1999-03-20`, by using this old date, this condition will return `false`, and the icons will be added. 

```JS
    const currentVersionInfo = await infoServices.getLatestVersionInfo()
    const currVersionDate = new Date(currentVersionInfo ? currentVersionInfo.createdAt : '1999-03-20')
    if (latestReleaseDate <= currVersionDate) {
        return
    }
```

Reply:
got it. makes sense.